src/homework/day11/UtilitaryClass.java

```diff
@@ -0,0 +1,30 @@
+package homework.day11;
+
+import java.util.Iterator;
+import java.util.List;
+
+public class UtilitaryClass {
+
+    public synchronized static void removeListElement(List<Mouse> mouseList) {
+        Iterator<Mouse> iterator = mouseList.listIterator();
+
+        while (iterator.hasNext()) {
+            iterator.next().peep();
+            iterator.remove();
+        }
+    }
+
+    public synchronized static void removeListElementsWitDelay(List<Mouse> mouseList) {
```

Comment:
Only FYI (not mandatory to fix)
probably, it's easy to create separate method sleep, because your 2 methods do pretty the same, difference only in sleep

Reply:
re-checked as a separate method - doesn't work